package.xml

```diff
@@ -38,7 +38,7 @@
   <license>New BSD</license>
   <url type="website">http://ceres-solver.org/</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
```

Comment:
Good point about not needing ament_cmake to build.  I guess I could remove that line, which would be a little more correct, but wouldn't actually make it any easier to keep in sync with upstream.

Just so I understand, why would it work to leave buildtool_depend as catkin?  Wouldn't ament look at that, try to satisfy the dependency, and then fail?